app/ui/about/newtab.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+- License, v. 2.0. If a copy of the MPL was not distributed with this
+- file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>New Tab</title>
+</head>
+<body>
+
+Tofino: <a href="https://github.com/mozilla/tofino">https://github.com/mozilla/tofino</a>
+<ul>
+  <li><a href="tofino://history">History</a></li>
+  <li><a href="tofino://stars">Stars</a></li>
+  <li><a href="tofino://saves">Saves</a></li>
+</ul>
+
+</body>
```

Comment:
Currently all `tofino://foobar` pages are plain HTML. We should stop doing this and be consistent across our application code and use React everywhere. I'm working on converting the old pages at the moment, so any new page should follow this.